### ISSUE TITLE
Default (HTML Required) validation triggered when usiong FileInput component

### ISSUE THREAD

**Issue by suparthghimire** [mantinedev/mantine#1900]
### What package has an issue

@mantine/core

### Describe the bug

When I use native HTML <input type="file"/> everything fine. But when I use <FileInput/> component from mantine, clicking on the input tag causes default HTML validation to get triggered.
Here is My Code: 
```ts
<form
          className={classes.form}
          onSubmit={(event) => {
            event.preventDefault();
            console.log("FORM SUBMITTED");
          }}
        >
          <Text size="lg" weight={700} className={classes.title}>
            Create New User
          </Text>

          <div className={classes.fields}>
            {inputs.map((input, index) => {
              return (
                <SimpleGrid
                  key={"create-user-field-" + index}
                  cols={input.fields.length}
                  breakpoints={[{ maxWidth: "sm", cols: 1 }]}
                >
                  {input.fields.map((field, index) => {
                    const fieldType = field.type;
                    if (fieldType === "select") {
                      return (
                        <Select
                          key={"create-user-field-input-" + index}
                          label="Select Country of User"
                          placeholder="Search Country"
                          required={field.required}
                          name={field.name}
                          searchable
                          data={field.options ? field.options : []}
                          className={classes.fieldInput}
                        />
                      );
                    } else if (fieldType === "file") {
                      return (
                        <div key={"create-user-field-input-" + index}>
                          <input type="file" name={field.name} id="" />
                          <FileInput />
                        </div>
                      );
                    } else if (fieldType === "text" || fieldType === "email") {
                      return (
                        <TextInput
                          type={field.type}
                          name={field.name}
                          key={"create-user-field-input-" + index}
                          label={field.label}
                          placeholder={field.placeholder}
                          required={field.required}
                          className={classes.fieldInput}
                        />
                      );
                    }
                  })}
                </SimpleGrid>
              );
            })}

            <Group position="right" mt="md">
              <Button type="submit" className={classes.control}>
                Send message
              </Button>
            </Group>
          </div>
        </form>
```

### In which browser did the problem occur

Chrome

**Comment by NithinSK04** [comment]
Even I'm facing the same issue !

**Comment by jintaokoong** [comment]
You can solve this issue by giving it a `type` attribute of value `button`.

**Comment by rtivital** [comment]
Fixed in 5.0.2
